### DISCUSSION TITLE
Cooperate Design Presentation Material available?

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1379] [Good to know]
Hi,
do we have some presentation material e.g. basic PowerPoint presentation in a StreamPipes cooperate design?
Or any material that was used before?

**Comment by bossenti** [comment]
@flomickl We have some slides pf previous talks online on our homepage: https://streampipes.apache.org/media.html
But as far as I know, there is no template slide deck

**Reply by dominikriemer** [reply]
Here is a presentation template:
https://cwiki.apache.org/confluence/display/STREAMPIPES/Presentations

Fonts used are listed here:
https://cwiki.apache.org/confluence/display/STREAMPIPES/Fonts
